pkg/kritis/metadata/containeranalysis/containeranalysis.go

```diff
@@ -294,3 +296,74 @@ func (c Client) DeleteOccurrence(ID string) error {
 	glog.Infof("executed deletion of occurrence=%s", ID)
 	return c.client.DeleteOccurrence(c.ctx, req)
 }
+
+// Poll discovery occurrence for an image and wait until container analysis
+// finishes. Throws an error if analysis is not successful or timeouts.
+func WaitForVulnzAnalysis(containerImage string, timeout time.Duration) error {
```

Comment:
Should this go to metadata.go?  It doesn't seem to be tied exclusively to the containeranalysis client.

Also, I suggest passing the client to the function, instead of creating it inside of it.